rocket/rocket.py

```diff
@@ -139,6 +148,12 @@ def _build_and_deploy(
             for file in extract_python_files_from_folder(package_dir):
                 files.append(file)
 
+        if isinstance(glob_path, str):
+            files.extend(glob.glob(os.path.join(project_location, glob_path)))
+        elif isinstance(glob_path, list):
+            for path in glob_path:
+                files.extend(glob.glob(os.path.join(project_location, path)))
```

Comment:
Let's use this glob file maybe also for modified files. We can add by default `.py`

Reply:
Can you specify where you would like to use glob, too? The place you are mentioning sounds like: 
- https://github.com/getyourguide/db-rocket/blob/e1840e48f7778863b577795a3fa176ddd85125c4/rocket/utils.py#L47
- https://github.com/getyourguide/db-rocket/blob/e1840e48f7778863b577795a3fa176ddd85125c4/rocket/utils.py#L33

In both cases I would recommend to not use glob. Walk is safer as it automatically moves arbitrarily deep into a nested directory structure. In glob you would need to do something like this: "*.py", "*/*.py", "*/*/*.py", ...